datasets/opus_openoffice/opus_openoffice.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OpusOpenoffice: A collection of documents from openoffice org"""
+
+from __future__ import absolute_import, division, print_function
+
+import itertools
+import os
+
+import datasets
+
+
+_CITATION = """\
+J. Tiedemann, 2012, Parallel Data, Tools and Interfaces in OPUS. \
```

Comment:
can you use the citation in bibtex format ?

Reply:
done @lhoestq